### DISCUSSION TITLE
How do I remove and add more pages on the top menu bar

### DISCUSSION THREAD

**Discussion by ShenjieZhouBAS** [academicpages/academicpages.github.io#2077] [Q&A] [answered]
I am currently trying to keep the website simple, so I am thinking about keeping the CV pages, and remove the rest of other pages such as blog posts, publication etc. I would like to have a page talking about my research highlight with links to paper and media coverage and some figures. Should I create a whole new page with different names or can I modify based on one of the pre-set pages?

**Comment by rjzupkoii** [comment, marked as answer]
The top navigation bar can be adjusted by changing the contents of `_data/navigation.yml`, please see the [Customizing the layout of your site](https://github.com/academicpages/academicpages.github.io/wiki/Customizing-the-layout-of-your-site#order-of-navigation-bar) Wiki page for more details. The Wiki is still a work in progress so we'll be adding more to it over the coming weeks.
